BrowserKit/Sources/ComponentLibrary/Headers/NavigationHeaderView.swift

```diff
@@ -40,8 +38,7 @@ public final class NavigationHeaderView: UIView {
                         for: .normal)
         button.titleLabel?.adjustsFontSizeToFitWidth = true
         button.addTarget(self, action: #selector(self.backButtonTapped), for: .touchUpInside)
-        button.setContentHuggingPriority(.defaultHigh, for: .horizontal)
-        button.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
+        button.contentHorizontalAlignment = .left
```

Comment:
Why `.left`? Wondering if that could be an issue with RTL languages. Maybe `.leading` would be the better choice.